tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react"
+  },
+  "include": [
+    "src"
+  ],
+  "files": [
+    "src/utils.d.ts"
+  ]
```

Comment:
Does the `include` not also grab `src/utils.d.ts`?

Reply:
Oh you're right. I didn't realize that. Fixed in https://github.com/iFixit/product-page-components/pull/31/commits/d17dca23b5fcf4f5ca50563f57f45cfbefa8fd81.